### DISCUSSION TITLE
Instance not getting connected to putty

### DISCUSSION THREAD

**Discussion by Maheshdvp** [learndevopsonline/learndevopsonline#1650] [Q&A] [answered]
Hi team, I had used this server previously.It had connected to putty fine also but now I cant able to connect same instance showing error connection time out. Not only that server no server is'nt able connect showing same error. I had double checked security group as well. Could you please help me on that.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/3faff7c7-fa07-4685-b8b5-1a14d45b0516)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/9d359b9a-7fcb-4632-ad7a-cd792c81b732)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/c538d73e-730a-42ce-98df-ef547fcb32f4)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/c01f00d6-84c6-4924-8600-5fcbf41f9bbe)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/3e357178-1701-4e85-8b6d-69df17e8b110)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/3900aea0-f9ca-490d-a2c3-b93a6a29511f)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/35233fe5-f611-454f-8c38-19edd0323776)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/c352cce8-d49f-40a4-86b3-2c6b2dad411d)

**Comment by sreedharm07** [comment]
are you using office laptop?

**Comment by Maheshdvp** [comment]
No, I am not.

**Comment by Maheshdvp** [comment]
Can I keep my Aws credentials to your mail

**Reply by sreedharm07** [reply]
yes please

**Comment by Maheshdvp** [comment]
Address please

**Reply by sreedharm07** [reply]
please use only one thread you are using multiple threads 
mail sreedhar70755@gmail.com

**Reply by Maheshdvp** [reply]
Done Please check

**Reply by sreedharm07** [reply, marked as answer]
@Maheshdvp 
bro please login now it will work 
for your route table internet gateway is not attached before i attached now thats the error

**Reply by Maheshdvp** [reply]
yes bro working fine Thank you
